packages/analyticsdx-template-lint/test/unit/linter/autoInstall.test.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+import * as path from 'path';
+import { ERRORS } from '../../../src';
+import { getDiagnosticsForPath } from '../../testutils';
+import { StringDocument, stringifyDiagnostics, TestLinter } from './testlinter';
+
+// tslint:disable: no-unused-expression
```

Comment:
Nit: I see these tslint comments in several of the test files but is there an unused expression in these tests?

Reply:
This is kind of boilerplate for chai's expect's, and I just copied it over. But, this file does have one on line 57 (`expect(...).to.be.undefined;`).
I think there's a way to configure tslint do have this on by default for test files and/or for chai, but I never got around to hooking it up. Plus, we should really move eslint at some point.